src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java

```diff
@@ -220,6 +220,7 @@ private String getNoticeBody(SscsCaseData caseData) {
                 case CORRECTION:
                     return caseData.getDocumentGeneration().getCorrectionBodyContent();
                 case STATEMENT_OF_REASONS:
+                    return caseData.getDocumentGeneration().getStatementOfReasonsBodyContent();
```

Comment:
Please confirm that this whole method getNoticeBody is behind a feature flag once you have rebased master

Reply:
done under https://github.com/hmcts/sscs-tribunals-case-api/pull/3244